### PR TITLE
Hw1

### PR DESCRIPTION
**Pull request by gleason9113**
Changes:

01-selectors.md
Added answers for each level of the CSS Diner game.

02-palindrome.html
Added bootstrap classes to HTML elements to format the document as shown in the example.

02-palindrome.js
Added handleInput function to parse text input, check for non-numeric characters and reversibility, then print the results to the webpage.

Feedback sought:  All feedback is welcome.  I am particularly interested in any input on the Bootstrap elements - I feel like I have a hard time with HTML/CSS styling.

### CODE REVIEW

**Comment by gleason9113** [comment]
This pull request doesn't show any file changes bc I originally committed to main.  I thought I had reverted that, but apparently not.

File: hw1/02-palindrome.js [hunk 1]
```diff
@@ -1,28 +1,3 @@
 const elem = document.querySelector('input');
 
 elem.addEventListener('input', handleInput);
```
Comment (rohan-singh1) [inline hw1/02-palindrome.js]: 'handleInput' was used before it was defined.

File: hw1/02-palindrome.js [hunk 2]
```diff
@@ -1,28 +1,3 @@
...
-
-function handleInput(event) {
-  const input = event.target.value;
-  let out = document.getElementById('answer')
```
Comment (rohan-singh1) [inline hw1/02-palindrome.js]: 'out' is never reassigned. Use 'const' instead.

File: hw1/02-palindrome.js [hunk 3]
```diff
@@ -1,28 +1,3 @@
...
-  let numCheck = new RegExp("^[0-9]+$"); //Test for non-numeric values in input
-  let output = '';
-  let pass = false; //Flag for output color
-  let reverse = input.split('').reverse().join(''); //Reverse the input string
```
Comment (rohan-singh1) [inline hw1/02-palindrome.js]: Expected exception block, space or tab after '//' in comment.

**Review by rohan-singh1 — COMMENTED** [review]
There are several linter issues I observed, but they are of only two types:

1. Usage of const wherever applicable.
2. Addition of a space after comment slashes.

I have pointed out two lines for each type. Apart from that, the code looks good. I especially liked your usage of Regex for input validation.

**Pull request merged**